client/my-sites/hosting/main.js

```diff
@@ -227,6 +228,12 @@ const Hosting = ( props ) => {
 
 	const trialRequested = () => {
 		setHasTransferring( true );
+		setShouldPoolTransferResults( true );
```

Comment:
should be `setShouldPollTransferResults` ?

Reply:
Updated, thanks!

Reply:
you currently have `setShouldPullTransferResults` instead of `setShouldPollTransferResults`

Reply:
Yes, the expected term here is `pull` since we're getting that data. Am I missing something?

Reply:
Cool, I thought it was polling the endpoint for the transfer state. but pull is good.